webclient/src/components/views/Create.vue

```diff
@@ -54,6 +59,7 @@ export default {
         })
         .then((project) => {
           router.push('/details/' + project.ID);
+          self.loading = true;
```

Comment:
What if it fails?

I know we aren't displaying an error which we should, but if it fails we should turn the spinner off to. Also, it might be better to flip these so we navigate after stopping the spinner.